Polyclinic/Polyclinic.AppHost/AppHost.cs

```diff
@@ -0,0 +1,14 @@
+﻿var builder = DistributedApplication.CreateBuilder(args);
+
+var mysql = builder.AddMySql("mysql");
+var mysqlDb = mysql.AddDatabase("mysqldb");
+
+var api = builder.AddProject<Projects.Polyclinic_Api>("polyclinic-api")
+    .WithReference(mysqlDb, "mysqldb")
+    .WaitFor(mysqlDb);
+
+builder.AddProject<Projects.Polyclinic_Grpc_Client>("polyclinic-grpc-client")
```

Comment:
Чтобы не создавались лишние endpoints можно просто указать чтобы Aspire не брал данные с LaunchProfile.
```suggestion
builder.AddProject<Projects.Polyclinic_Grpc_Client>("polyclinic-grpc-client", launchProfileName: null)
```